common/SmokeTests/SmokeTest/Update-Dependencies.ps1

```diff
@@ -0,0 +1,114 @@
+param(
+    $ProjectFile = './SmokeTest.csproj',
+    $FeedName = 'NightlyFeed'
+)
+
+$PACKAGE_EXCLUSIONS = @{
+    'Azure.Security.Keyvault.Secrets' = @{
+        '4.1.0-dev.20191102.1' = $true;
```

Comment:
I assume we cannot easily clean these off the feed?

Reply:
I can clean these off the feed using `sleet delete`. Our concern with the exclusion list was that we would encounter higher version packages in situations like the package csproj having higher version numbers in `master` or development on another branch publishing higher version numbers. 

These are the packages that interfered because their version numbers sorted later in the sequence because of major and minor numbers.

Reply:
These have been removed.